var/spack/repos/builtin/packages/micromamba/package.py

```diff
@@ -45,8 +45,8 @@ class Micromamba(CMakePackage):
         depends_on("yaml-cpp", type="link")
         depends_on("libreproc+cxx+shared", type="link")
         depends_on("tl-expected@2022-11-24", type="link")
-        depends_on("fmt", type="link")
-        depends_on("spdlog", type="link")
+        depends_on("fmt@9.1.0", type="link")
+        depends_on("spdlog@1.11.0", type="link")
```

Comment:
Is it intentional to constrain these packages to those versions?  Or would you want those as the minimum versions?

Reply:
I guess, these could be the minimum versions. I suggested these as I could test these version combinations on three different systems, which worked for me. (Omitting the details)

Reply:
This can always be dealt with later if/when it becomes an issue.